### PR TITLE
Add PersonaPlex / Moshi full-duplex S2S support (Mimi codec + Moshi LM + ORT example)

### PR DESCRIPTION
**Pull request by justinchuby**
Adds end-to-end ONNX support for [`nvidia/personaplex-7b-v1`](https://huggingface.co/nvidia/personaplex-7b-v1), the Kyutai **Moshi** full-duplex speech-to-speech architecture, built declaratively with `onnxscript.nn` from the native Kyutai `safetensors` checkpoints (no HF `config.json`).

Three phases, one PR:

### P1 — Mimi neural codec (commit `0fc84c1`)
SEANet encoder/decoder + split-RVQ + codec transformer. Encoder codes exact, decode ~2e-7 vs reference. Root cause fixed: Kyutai LayerNorm eps is `1e-5` (mobius default was `1e-6`).

### P2 — Moshi LM (temporal + depformer)
- `models/moshi.py`:
  - **MoshiTemporalModel** — dim=4096, 32L, 32H, RoPE θ=1e4 interleaved, sliding-window causal (3000), SwiGLU via `FusedGateUpMLP`, RMSNorm eps=1e-8.
  - **MoshiDepformerModel** — dim=1024, 6L, no RoPE, full causal, `weights_per_step=16` per-substep linears; emitted as a **one-substep graph** selected by `substep_index` (Gather), embedding select via `Where` (no `If`), looped 16× externally.
- `tasks/_moshi.py`: `MoshiTemporalTask` (`input_frame[B,17,S]` + KV → hidden + text_logits + KV) and `MoshiDepformerTask` (`hidden` + `prev_token` + `substep_index` + KV → logits + KV).
- `integrations/moshi`: `build_moshi_lm()` native loader → `{"temporal", "depformer"}` ModelPackages.
- **Parity:** exact argmax match vs the Kyutai reference (CPU exact; CUDA exact with `use_tf32=0` — H200 TF32 otherwise flips greedy sampling). Committed golden + generator script; integration parity test (CPU-only) **passes**.

### P3 — ONNX Runtime example
- `examples/personaplex_moshi.py`: faithful NumPy port of Kyutai `LMGen.step` — ring cache with per-codebook delays `[0,0,1,1,1,1,1,1,1,0,1,1,1,1,1,1,1]`, temporal step, greedy text sampling, 16-substep autoregressive depformer, delayed output collection — driving four ORT sessions (Mimi encoder/decoder + temporal + depformer) for full-duplex S2S. Verified end to end (5 input frames → 3 assistant frames → decoded waveform).

## Testing
- L1 build tests for temporal + depformer pass; full moshi/mimi/codec L1 suite green (9 tests).
- Integration parity test passes (`tests/moshi_lm_integration_test.py`, ~280s, CPU).
- Example verified end-to-end (build + generate + Mimi decode → `assistant.wav`).
- Lint clean on new files.

> Note: the temporal/depformer reach mobius only via the native `build_moshi_lm` loader (not the standard `build(model_id)` registry path), consistent with the Mimi codec which also uses a native Kyutai-format loader.

Co-authored-by: Copilot <223556219+Copilot@users.noreply.github.com>

### CODE REVIEW

File: examples/personaplex/README.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+## 2. Run the browser server (needs a CUDA GPU for real time)
+
+The server only loads ONNX models, so it can run in a lightweight
+`onnxruntime-gpu` environment without `mobius`:
+
+```bash
+pip install aiohttp onnxruntime-gpu numpy sentencepiece huggingface_hub
+python examples/personaplex/server.py \
+    --model-dir output/personaplex/onnx --device cuda \
+    --host 0.0.0.0 --port 7681
```
Comment (justinchuby) [inline examples/personaplex/README.md]: Use local host instead?
